Frameworks/Core/ERPrototypes/Resources/erprototypes.eomodeld/EOJDBCPostgresqlPrototypes.plist

```diff
@@ -359,6 +359,13 @@
             valueClassName = NSString; 
             valueType = S; 
             width = 10000000; 
+        }, 
+        {
+            columnName = uuid; 
+            externalType = bytea; 
```

Comment:
Hi Samuel,

Would it be possible to use [PostgreSQL's `uuid`](https://www.postgresql.org/docs/current/static/datatype-uuid.html) as the external type here? Or does it create problems elsewhere?